babel-preset/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "babel-preset-react-native",
+  "version": "1.1.0",
+  "description": "Babel preset for React Native applications",
+  "main": "index.js",
+  "repository": "https://github.com/facebook/react-native/tree/master/babel-preset",
+  "keywords": [
+    "babel",
+    "preset",
+    "react-native"
+  ],
+  "license" : "BSD-3-Clause",
+  "bugs": {
+    "url": "https://github.com/facebook/react-native/issues"
+  },
+  "homepage": "https://github.com/facebook/react-native/tree/master/babel-preset/README.md",
+  "dependencies": {
+    "babel-plugin-react-transform": "2.0.0-beta1",
```

Comment:
will we need to land the pr to update babel 6.3.x first, right? I think that one hasn't been merged internally. cc @davidaurelio